packages/core/cli/README.md

```diff
@@ -1,44 +1,269 @@
 # Vue Storefront CLI
-## Commands
-### Init new project
+
+<!-- toc -->
+* [Usage](#usage)
+* [Commands](#commands)
+<!-- tocstop -->
+# Usage
+<!-- usage -->
+```sh-session
+$ npm install -g @vue-storefront/cli
+$ @vue-storefront/cli COMMAND
```

Comment:
What's the motivation to use `@vue-storefront/cli` as the name for the tool? Maybe that can be used `vue-storefront` to reinforce the brand name, just a suggestion :)

```suggestion
vue-storefront COMMAND
```

Reply:
Oh, and nice work on documentation, it's really informative!

Reply:
I don't deserve your compliment. The `oclif readme` command automatically generated it.

Reply:
But about the name `@vue-storefront/cli`:

It's backward compatible with the current CLI and with the marketing materials to quickly share how to start a store with VSF.